### ISSUE TITLE
1.13 Release

### ISSUE THREAD

**Issue by suchipi** [prettier/prettier#4444]
It's getting to be time :)

I can probably put release notes together this weekend, if we're ready. I'll create a milestone and put a couple PRs in it.

**Comment by lipis** [comment]
I also added the #4440 to 1.13, I hope it's OK :)

**Comment by suchipi** [comment]
Totally, thanks @lipis

**Comment by kachkaev** [comment]
Would it be possible for you to release `1.13.0-beta` from the latest master so that it contains #4192 and #4341? @robwise could easily test editor integration in https://github.com/prettier/prettier-atom/pull/404 in this case.

I could also cut a new unofficial version of Prettier as I [once did earlier](https://github.com/prettier/prettier/issues/4000#issuecomment-376119478), but it'd be probably better if it was ‘official’. There is a past record of beta/RCs for Prettier package, although these were not very frequent:

<img width="645" alt="screen shot 2018-05-09 at 20 08 17" src="https://user-images.githubusercontent.com/608862/39834639-8ae1e91a-53c5-11e8-87be-30009786cbd4.png">

https://www.npmjs.com/package/prettier (tab `versions`)

**Comment by suchipi** [comment]
I can, but would the latest master from git work?
```json
  "dependencies": {
    "prettier": "prettier/prettier#92485601eb3be9c14d2e38235d3400b61be8740a"
  },
```

**Comment by kachkaev** [comment]
Do you want me to do this? The result will be `kachkaev/prettier#HASH`, which may look a bit smelly for those who want to install this on their machine 😅

**Comment by duailibe** [comment]
@kachkaev You can run `yarn build && cd dist/ && npm pack` and install the tarball 😅

**Comment by suchipi** [comment]
@kachkaev I think that's fine, the smell is present because it's an unreleased feature, the smell will go away once 1.13.0 is released 😉

**Comment by kachkaev** [comment]
I’m just saying that an official beta would look more familiar and safe than something starting with `kachkaev/...` (especially for those who don’t know me, i.e. everyone haha).

I thought that releasing a beta would be as easy as:

1. git checkout master
1. git tag
2. yarn build && cd build
4. npm publish --tag=beta

It’s not rare in NPM communities to release betas and RCs, mainly to reduce the number of bug reports shorty after the official release. Tests are great, but tests + some real work in the wild is better :–)

In addition to helping @robwise, a beta would allow me to update install docs in prettier-plugin-elm, where I can finally suggest installing Prettier globally. My hand-crafted dist fork will do the same, but it will look slightly scarier.

No worries if cutting betas is not a priority for Prettier these days – other tasks may be indeed more fruitful. However, it’d be great if pre-releases became a common practice for Prettier communify in future ;–) That’s not a waste of time with over 1M downloads per week for your awesome tool!

**Comment by lipis** [comment]
It's a bit more than a 4 step process :)
https://github.com/prettier/prettier/wiki/Release-Checklist

**Comment by lipis** [comment]
But of course.. some of them do not apply to beta..

**Comment by j-f1** [comment]
Looks like only ~10 of them apply.

**Comment by kachkaev** [comment]
@robwise feel free to `npm install github:kachkaev/prettier#1.13.0-beta.0` to play with the upcoming Prettier features in https://github.com/prettier/prettier-atom/pull/404 🙌 

__Package source:__ https://github.com/kachkaev/prettier/tree/1.13.0-beta.0  
__Built from:__ Prettier's master at https://github.com/prettier/prettier/commit/6447d30b0b8e5bd2691c9a6cfc9c0337ef4ae2f2

🤷‍♂️

**Comment by vjeux** [comment]
Could we get an update of babel and flow in as well?

**Comment by duailibe** [comment]
@vjeux I think Babylon update is blocked on node 4 support

**Comment by suchipi** [comment]
@duailibe if we merged your bundle stuff, we could compile babylon, right?

**Comment by duailibe** [comment]
@suchipi Yes.. we would only stop supporting node 4 for installing directly from github

**Comment by duailibe** [comment]
:+1: Gonna do it

**Comment by vjeux** [comment]
If babylon requires node 4, then I think that we have to take the plunge and drop node 4 support.

**Comment by vjeux** [comment]
Babylon and Flow upgrade PR: #4536

**Comment by vjeux** [comment]
Okay, everything is good on my side :)

**Comment by duailibe** [comment]
Everything on the milestone is done https://github.com/prettier/prettier/milestone/16 and all the PRs are detailed in the blog post PR.

I'll just wait for someone to give a final look on the blog post and I'll release

**Comment by duailibe** [comment]
Done!
